templates/connect.php

```diff
@@ -414,10 +406,7 @@ class="button button-secondary" tabindex="2"><?php fs_esc_html_echo_x_inline( 'S
 			// Set core permission list items.
 			$permissions = array();
 
-			// Add newsletter permissions if enabled.
-			if ( $is_gdpr_required || $fs->is_permission_requested( 'newsletter' ) ) {
-				$permissions[] = $permission_manager->get_newsletter_permission();
-			}
+            $permissions[] = $permission_manager->get_newsletter_permission();

```

Comment:
@fajardoleo let's include it only when `$fs->is_permission_requested( 'newsletter' )`